_health-care/_js/_form.js

```diff
@@ -0,0 +1,348 @@
+    { node: "ChildNameFirst",  arg: "FirstChild2", transform: identityTransform },
+    { node: "ChildNameMiddle",  arg: "MiddleChild2", transform: identityTransform },
+    { node: "ChildNameSuffix",  arg: "II", transform: identityTransform },
+    { node: "ChildRelationshipSon",  arg: "0", transform: identityTransform },
+    { node: "ChildRelationshipDaughter",  arg: "1", transform: identityTransform },
+    { node: "ChildRelationshipStepson",  arg: "0", transform: identityTransform },
+    { node: "ChildRelationshipStepdaughter",  arg: "0", transform: identityTransform },
+    { node: "ChildSocialSecurityNumber",  arg: "555-44-3333", transform: identityTransform },
+    { node: "ChildDependentDateMonth",  arg: "4", transform: identityTransform },
+    { node: "ChildDependentDateDay",  arg: "5", transform: identityTransform },
+    { node: "ChildDependentDateYear",  arg: "1992", transform: identityTransform },
+    { node: "ChildDependentDate",  arg: "04/05/1992", transform: identityTransform },
+    { node: "ChildDateOfBirthMonth",  arg: "4", transform: identityTransform },
+    { node: "ChildDateOfBirthDay",  arg: "5", transform: identityTransform },
+    { node: "ChildDateOfBirthYear",  arg: "1992", transform: identityTransform },
+    { node: "ChildDateOfBirth",  arg: "04/05/1992", transform: identityTransform },
+    { node: "ChildTotalDisabledYes",  arg: "0", transform: identityTransform },
+    { node: "ChildTotalDisabledNo",  arg: "1", transform: identityTransform },
+    { node: "ChildAttendSchoolYes",  arg: "1", transform: identityTransform },
+    { node: "ChildAttendSchoolNo",  arg: "0", transform: identityTransform },
+    { node: "ChildSelfSchool",  arg: "9876.00", transform: identityTransform },
+    { node: "ChildSupportAmount",  arg: null, transform: identityTransform },
+    { node: "ChildLive",  arg: "Y", transform: identityTransform },
+    { node: "ChildLiveNo",  arg: "0", transform: identityTransform },
+    { node: "ChildProvideSupportYes",  arg: "0", transform: identityTransform },
+    { node: "ChildProvideSupportNo",  arg: "0", transform: identityTransform },
+    { node: "ChildGrossIncome",  arg: "54.00", transform: identityTransform },
+    { node: "ChildPropertyIncome",  arg: "43.00", transform: identityTransform },
+    { node: "ChildOtherIncome",  arg: "32.00", transform: identityTransform }
+  ];
+
+  var getFormRoot = function() {
+    return document.querySelector(".main-form");
+  };
+
+  var saveForm = function(formRoot, e) {
+    sessionStorage["voa_form"] = JSON.stringify($(formRoot).serializeArray());
+  };
+
+  var initForm = function() {
+    var formRoot = getFormRoot();
+
+    // Suppress the submit behavior since submission is done via crafting an
+    // XML payload for AJAX.
+    formRoot.addEventListener("submit",
+        function (e) {
+          e.preventDefault();
+          console.log(build1010ezXml(formRoot));
```

Comment:
Add a TODO.  BTW, I am LOVING the thoroughness of your review here. 

Like seriously <3-ing.

Thank you!!!!!!